.golangci.yml

```diff
@@ -111,8 +111,16 @@ linters-settings:
   gofumpt:
     extra-rules: true
   perfsprint:
-     # Optimizes `fmt.Errorf`.
+    # Optimizes even if it requires an int or uint type cast.
+    int-conversion: true
+    # Optimizes into `err.Error()` even if it is only equivalent for non-nil errors.
+    err-error: true
+    # Optimizes `fmt.Errorf`.
     errorf: true
+    # Optimizes `fmt.Sprintf` with only one argument.
```

Comment:
Thanks for this.
I personally find sprintf more readable.
I see most changes only involve tests, unless we can identify hot paths where concatenation would make a difference, I'd personally stick to sprintf

Could you provide examples for the other rules?